cibuildwheel.toml

```diff
@@ -0,0 +1,91 @@
+[tool.cibuildwheel]
+
+# Don't build 32-bit wheels
+skip = [
+  "*-manylinux_i686",
+  "*-musllinux_i686",
+  "*-win32",
+]
+
+# Apple Silicon support
+# When cross-compiling on Intel, it is not possible to test arm64 and
+# the arm64 part of a universal2 wheel. Warnings will be silenced with
+# following CIBW_TEST_SKIP
+macos.archs = ["x86_64", "arm64", "universal2"]
+test-skip = "*-macosx_arm64 *_universal2:arm64"
+
+build-verbosity = 1
+
+linux.before-all = [
+  "yum install -y llvm-toolset-7.0 || yum -v install -y llvm-toolset-7.0",
+]
+
+# lxml doesn't have a wheel for Python 3.10 on the manylinux image we use.
+# lxml has historically been slow to support new Pythons as well.
```

Comment:
I don't understand this. `lxml` has 3.11 wheels and 3.10 wheels and they are manylinux2010 and this is 2014 afaict (unless you are on an old cibuildwheel but then it's still 2010 at least). Still needed?

Reply:
Nope, this is out of date :-)

Reply:
I'm giving it a try without it, then! :)